server/BraveAlerterConfigurator.js

```diff
@@ -96,12 +75,12 @@ class BraveAlerterConfigurator {
       session = await db.getSessionWithSessionId(alertSession.sessionId, pgClient)
 
       if (session) {
-        // If this is not a OneSignal session (i.e. we are given a respondedByPhoneNumber) and the session has no respondedByPhoneNumber, then this is the first SMS response, so assign it as the session's respondedByPhoneNumber
+        // If the session has no respondedByPhoneNumber, then this is the first SMS response, so assign it as the session's respondedByPhoneNumber
         if (alertSession.respondedByPhoneNumber !== undefined && session.respondedByPhoneNumber === null) {
           session.respondedByPhoneNumber = alertSession.respondedByPhoneNumber
         }
 
-        // If this is a OneSignal session (i.e. it isn't given the respondedByPhoneNumber) or if the SMS came from the session's respondedByPhoneNumber
+        // If the SMS came from the session's respondedByPhoneNumber
         if (alertSession.respondedByPhoneNumber === undefined || alertSession.respondedByPhoneNumber === session.respondedByPhoneNumber) {
```

Comment:
Similarly, I think you can leave out the `alertSession.respondedByPhoneNumber === undefined` part of this condition because now that it will always be an SMS, `alertSession.respondedByPhoneNumber` should never be `undefined`.